tests/Vue/Components/MismatchRow.spec.js

```diff
@@ -275,7 +275,7 @@ describe('MismatchesRow.vue', () => {
 
         await wrapper.find('.full-description-button').trigger('click');
 
-        const dialog = wrapper.find('.full-description-dialog .cdx-dialog');
```

Comment:
everything is working flawlessly, but how is this related to the compat build??? :thinking:

Reply:
TBH, I have no idea. The old one works with compat build, but needed to update to the new one to make it work without compat build.